api.go

```diff
@@ -41,6 +41,12 @@ type BlockAPI interface {
 	GetBlockByNumber(num int64) (*Block, error)
 }
 
+// AddressAPI provides address information
+type AddressAPI interface {
```

Comment:
```suggestion
type UTXOAddressAPI interface {
```

I would prefer to be specific, to avoid confusion